### DISCUSSION TITLE
uninstall pushkin

### DISCUSSION THREAD

**Discussion by Arnovia** [l3atbc-datadog/pushkin#181] [Q&A] [answered]
Hi,

Not giving up on pushkin at all, but I would like to remove it from one computer and want to make sure I do this right.
What is the cleanest way to fully uninstall pushkin and dependencies?

Thank you.
Best,

**Comment by jkhartshorne** [comment, marked as answer]
'pushkin armageddon' should delete your local website. 
'npm uninstall -g pushkin' should delete the pushkin tools
for uninstalling npm (assuming you want to) and docker, check their
documentation
finally, delete the directory with your website

that should delete most if not everything.

On May 11, 2021, Sebastian Waz ***@***.***> wrote:
> Hi,
>
> Not giving up on pushkin at all, but I would like to remove it from
> one computer and want to make sure I do this right.
>  What is the cleanest way to fully uninstall pushkin and dependencies?
>
> Thank you.
>  Best,
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/pushkin-consortium/pushkin/discussions/181>, or
unsubscribe <https://github.com/notifications/unsubscribe-
auth/ABOVVY3ZQAGUHJHFYKZIU5LTNEUO7ANCNFSM44VORWXA>.

**Reply by Arnovia** [reply]
Thank you.
